.github/dependabot.yml

```diff
@@ -10,4 +7,6 @@ updates:
   - package-ecosystem: "swift" # For Swift package updates
     directory: "/" # Location of Package.swift file
     schedule:
-      interval: "daily"
\ No newline at end of file
+      interval: "daily"
+    ignore:
+      - dependency-name: "GRDB"
```

Comment:
@Bunn do you know if this is a package name or repo name? Because while package name is `GRDB`, repo name is `GRDB.swift` (https://github.com/duckduckgo/GRDB.swift)

Reply:
Honestly, I'm not sure, the [docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore) are not very specific: `Use to ignore updates for dependencies with matching names`

Since it only says "name" and not "repo name" I'm assuming it's just `GRDB` 
If it doesn't work we can update to `GRDB*` or `GRDB.swift`